.devcontainer/conf/nginx/default.conf

```
@@ -0,0 +1,15 @@
+server {
+    listen 0.0.0.0:80 default_server;
+    root /var/www/public;
+    location / {
+        index index.php index.html;
+    }
+    location ~ \.php$ {
+        try_files $uri /index.php =404;
+        fastcgi_pass php:9000;
+        include fastcgi_params;
+        fastcgi_index index.php;
+        fastcgi_param SCRIPT_FILENAME $document_root/$fastcgi_script_name;
+        fastcgi_param PATH_INFO $fastcgi_path_info;
+    }
+}
```

To match offical documentation (https://glpi-install.readthedocs.io/en/latest/prerequisites.html#nginx-configuration)
```suggestion
server {
    listen 0.0.0.0:80 default_server;

    server_name glpi.localhost;

    root /var/www/public;

    location / {
        try_files $uri /index.php$is_args$args;
    }

    location ~ \.php$ {
        try_files $uri /index.php =404;
        fastcgi_pass unix:/run/php/php-fpm.sock;
        include fastcgi_params;
        fastcgi_index index.php;
        fastcgi_param SCRIPT_FILENAME $document_root/$fastcgi_script_name;
        fastcgi_param PATH_INFO $fastcgi_path_info;
    }
}
```